docs/09-Data.md

```diff
@@ -35,9 +32,40 @@ Use <a href="http://codeception.com/docs/06-ModulesAndHelpers#Dynamic-Configurat
 to set the database credentials from environment variables or from application configuration files.
 </div>
 
-After you enable this module in your test suite, it will automatically populate the database from a dump
-and repopulate it on each test run. These settings can be changed through the `populate` and `cleanup` options,
-which may be set to `false`.
+Db module can cleanup database between tests by loading a database dump. This can be done by parsing SQL file and 
+executing its commands using current connection
+
+```yaml
+modules:
+    config:
+        Db:
+            dsn: 'PDO DSN HERE'
+            user: 'root'
+            password:
+            dump: tests/_data/your-dump-name.sql
+            cleanup: true # reload dump between tests
+            populate: true # load dump before all tests
+            
+```
+
+ Alternatively an external tool (like mysql client, or pg_restore) can be used. This approach is faster and won't produce parsing errors while loading a dump.
+ Use `populator` config option to specify the command. For MySQL it can look like this:
+ 
+```yaml
+ modules:
+    enabled:
+       - Db:
+          dsn: 'mysql:host=localhost;dbname=testdb'
+          user: 'root'
+          password: ''
+          cleanup: true # run populator before each test
+          populate: true # run populator before all test
+          populator: 'mysql -u $user $dbname < tests/_data/dump.sql'
+```
+
+See the []Db module reference](http://codeception.com/docs/modules/Db#SQL-data-dump) for more examples.
+
+To ensure database dump is loaded before all tests add `populate: true`. To clean current database and reload dump between tests use `cleanup: true`.
```

Comment:
👍 to add this description of the behavior expected. This has never been documented and now the intention is clear.